package grupo3.desafioFinalBootcamp.exceptions;

public class DuplicateFlightNumber extends Exception {
    public final String ERROR = "El n√∫mero de vuelo ya se encuentra utilizado.";

    public DuplicateFlightNumber() {
        super();
    }
}
